src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -162,38 +191,82 @@ public void setDesiredPower(double power) {
     m_desiredPower = power;
   }
 
+  /**
+   * Reset/zero the inbuilt falcon motor encoder
+   */
   public void zeroEncoder() {
     m_motor.setSelectedSensorPosition(0.0);
   }
 
+  /**
+   * If we are not calibrated and we are trying to set the elevator mode to 
+   * something other than CALIBRATION Or DISABLED, don't do anything.
+   * 
+   * If we are calibrated, set the mode to whatever we desire. 
+   * 
+   * @param mode
+   */
   public void setMode(ElevatorMode mode) {
     if (!m_isCalibrated && !(mode == ElevatorMode.CALIBRATION || mode == ElevatorMode.DISABLED)) return;
     m_mode = mode;
     if (m_mode == ElevatorMode.POSITION)
       DrawMechanism.getInstance().setElevatorExtension(m_desiredPosition);
   }
 
+  /**
+   * Get the velocity of the elevator in meters per seocnd
+   * @return elevator velocity
+   */
   public double getVelocity() {
     return Conversions.falconToMPS(
       m_motor.getSelectedSensorVelocity(),
       ElevatorConstants.kSpoolCircumference,
       ElevatorConstants.kGearRatio
     );
   }
+  
+  /**
+   * Calculate whether or not the current elevator position is within a set range of the desired position and that
+   * the velocity/speed of the elevator is less than a set velocity tolerance. 
+   * @return true or false-- whether or not the elevator has reached it's desired position
 
+   */
   public boolean reachedDesiredPosition() {
     return Math.abs(m_desiredPosition - getPosition()) < ElevatorConstants.kPositionTolerance
           && Math.abs(getVelocity()) < ElevatorConstants.kVelocityTolerance;
   }
 
+  /*
+   * Create enums for the different elevator positions:
+   *    Bottom without cone
+   *    Bottom with cone
+   *    Top without cone
+   *    Top with cone
+   *    None
+   */
+
   enum ElevatorStatus {
     BOTTOM, BOTTOM_CONE, TOP, TOP_CONE, NONE
   }
-
+  
+  /* 
+   * Create enums for the elevator mode, whether we want the elevator to calibrate, 
+   * use manual control, position control, or if we want the elevator to be disabled 
+   */
   public enum ElevatorMode {
     CALIBRATION, MANUAL, POSITION, DISABLED
   }
 
+  /**
+   * If our elevator position is less than the distance that the carraige can travel in the first stage, 
+   * check whether hasCone is true or false. If hasCone is true, set m_status to the ElevatorStatus enum BOTTOM_CONE. 
+   * If hasCone is false, set m_status to the ElevatorStatus enum BOTTOM. 
+   * 
+   * <p>
+   * 
+   * If our elelevator position is more than the distance that the carriage can travel in the first stage, check whether hasCone is
+   * true or false. If hasCone is true, set m_status to the ElevatorStatus enum TOP_CONE. If it's false, set m_status to the ElevatorStatus enum TOP.
+  */
```

Comment:
I think these comments should be in the function itself. Those kinds of comments explain how something happens. The javadoc can be as simple as "updates the elevator status based on the position and held game piece".